Makefile

```diff
@@ -9,11 +9,3 @@ image:
 test:
 	go test -race ./...
 .PHONY: test
-
-test-integration: build
-	./test/integration.sh
-.PHONY: test-integration
-
-vendor:
-	glide update --strip-vendor --skip-test
-.PHONY: vendor
```

Comment:
You should be replacing this with `go mod vendor`, not removing it

Reply:
Actually, icy ou're going to change this also fix go build to use `-mod vendor`

Reply:
Updated. It should be mentioned that there are no jobs set up to verify vendor, but that is not the scope of the PR. I just removed the two targets because they were irritating.